### PR TITLE
[OR-1470] change wton to wnativetoken

### PR DESCRIPTION
**Pull request by rlgns98kr**
- add deploy config (nativeTokenName, nativeTokenSymbol)
- change wton to wnativetoken
- add L1devnet in verify-scripts and fix typo and change version of compiler of WNativeToken

image

Thank you!

### CODE REVIEW

**Comment by rlgns98kr** [comment]
@nguyenzung 
I'm sorry. I accidentally did something else while I was working.
Legacy_ERC20_ETH -> Legacy_ERC20_{NativeTOkenName} will be processed in the next ticket. https://onther.atlassian.net/browse/OR-1415?atlOrigin=eyJpIjoiZGYwNjk3Mzc2YTg3NGQwYjgwODliNjM5NzBiMDU3NzAiLCJwIjoiaiJ9

File: packages/tokamak/contracts-bedrock/scripts/Deploy.s.sol [hunk 1]
```diff
@@ -145,7 +145,7 @@ contract Deploy is Deployer {
     /// @notice Deploy the Safe
     function deployL2NativeToken() public onlyDevnet broadcast {
         address addr_ = getL2NativeToken();
-        cfg.setNativeTokenAddress(addr_);
+        cfg.setNativeTokenAddress(addr_, string.concat(vm.projectRoot(), "/deploy-config/", deploymentContext, ".json"));
```
Comment (nguyenzung) [inline packages/tokamak/contracts-bedrock/scripts/Deploy.s.sol:148]: This function only works for devnet with the modifier onlyDevnet. I think we maybe should update this function to work with deploymentContext.

Comment (rlgns98kr) [reply packages/tokamak/contracts-bedrock/scripts/Deploy.s.sol:148]: Thank you!
I forgot to explain!
This function is to change the nativeTokenAddress of devnetl1.json to the actually distributed address. (Currently, the address set in the template goes to devnetL1.json.)
I think this should only work on devnet. 🙇🏼‍♂️

Comment (nguyenzung) [reply packages/tokamak/contracts-bedrock/scripts/Deploy.s.sol:148]: Thank you very much!

I understood.

**Review by nguyenzung — APPROVED** [review]
Great job, Steven :100:

**Pull request merged**